src/crypto/sign.js

```diff
@@ -141,7 +153,7 @@ export function verifyMessageWithTwoPublicKeys(signedMessage, publicKey, secondP
  * @return {string}
  */
 
-export function printSignedMessage(message, signedMessage, publicKey) {
+export function printSignedMessage({ message, signature, publicKey }) {
```

Comment:
This function should also handle second-signed messages, right?

Reply:
Not yet, this should be addressed for the future. I will create an issue for it